pennylane/math/__init__.py

```diff
@@ -69,6 +69,17 @@
 from .is_independent import is_independent
 
 sum = ar.numpy.sum
+
+
+def add(*args, **kwargs):
```

Comment:
```suggestion
def add(*args, **kwargs):
    """Add arguments element-wise."""
```

Maybe we should move this function to the `multi_dispatch.py` file.